### PR TITLE
update import statement order

### PR DESCRIPTION
**Pull request by thinkerou**
run the command: `goimports -m .`

### CODE REVIEW

File: pkg/cloudevents/transport/nats/transport.go [hunk 1]
```diff
@@ -3,10 +3,10 @@ package nats
 import (
 	"context"
 	"fmt"
+
 	"github.com/cloudevents/sdk-go/pkg/cloudevents"
 	context2 "github.com/cloudevents/sdk-go/pkg/cloudevents/context"
 	"github.com/cloudevents/sdk-go/pkg/cloudevents/transport"
-	"github.com/nats-io/go-nats"
```
Comment (n3wscott) [inline pkg/cloudevents/transport/nats/transport.go]: this was deleted and I think it is needed, this is why the CI is breaking.

**Comment by n3wscott** [comment]
Thanks a ton for the cleanup, it looks like one import was dropped that is breaking the CI. Add that back in and we can be in business :D

**Comment by thinkerou** [comment]
@n3wscott fixed! thanks!

**Comment by n3wscott** [comment]
LGTM, nice work, I use goland and never really see the imports unless I expand them.

**Comment by thinkerou** [comment]
@n3wscott maybe you should configure import set. because I use vim and vscode.

**Pull request merged**